packages/redux/src/contents/selectors.ts

```diff
@@ -321,3 +327,117 @@ export const getSEOMetadataResult = (
  */
 export const getContent = <T>(state: StoreState, hash: string) =>
   getEntityById(state, 'contents', hash) as ContentsEntity<T> | undefined;
+
+/**
+ * Returns the error thrown to the getSEOFiles request.
+ *
+ * @example
+ * ```
+ * import { getSEOFilesError } from '@farfetch/blackout-redux/contents';
+ *
+ * const mapStateToProps = (state, { query }) => ({
+ *     seoFilesError: getSEOFilesError(state, query)
+ * });
+ *
+ * ```
+ *
+ * @param state - Application state.
+ * @param query - Query applied to search the contents.
+ *
+ * @returns - Content error.
+ */
+export const getSEOFilesError = (
+  state: StoreState,
+  query: GetSEOFilesQuery,
+) => {
+  const hash = generateSEOFilesHash(query);
+  const error = getSEOFiles(state.contents as ContentsState).error;
+
+  return error && error[hash];
+};
+
+/**
+ * Returns the loading status to the getSEOFiles request.
+ *
+ * @example
+ * ```
+ * import { isSEOFilesLoading } from '@farfetch/blackout-redux/contents';
+ *
+ * const mapStateToProps = (state, { query }) => ({
+ *     isSEOMetadataLoading: isSEOFilesLoading(state, query)
+ * });
+ *
+ * ```
+ *
+ * @param state - Application state.
+ * @param query - Query applied to search the contents.
+ *
+ * @returns - If the content is loading or not.
+ */
+export const isSEOFilesLoading = (
+  state: StoreState,
+  query: GetSEOFilesQuery,
+) => {
+  const hash = generateSEOFilesHash(query);
+
+  return getSEOFiles(state.contents as ContentsState).isLoading[hash];
```

Comment:
```suggestion
  return !!getSEOFiles(state.contents as ContentsState).isLoading[hash];
```

Reply:
What is the purpose of `!!` where?

Reply:
To coerce the value to a boolean. `areSEOFilesLoading` is a selector that should return `true/false` and not `true/undefined`, which would be the case if the hash is not in state.

Reply:
Resolved! ✅